assemblies/assembly-configuring-postgresql-databases.adoc

```diff
@@ -0,0 +1,21 @@
+[id='assembly-configuring-postgresql-databases_{context}']
+= Configuring external PostgreSQL databases
+
+As an administrator, you can configure and use external PostgreSQL databases in {product}. You must have a PostgreSQL certificate file that is used to configure the {product-very-short} instance either using the Operator or Helm Chart.
+
+[NOTE]
+====
+{product-very-short} supports only configuring PostgreSQL databases. You are responsible for maintenance activities, such as backing up your data or configuring high availability (HA) for the configured PostgreSQL databases.
+====
```

Comment:
It looks like a question for PM, I'd say it is the point, yes.

Reply:
If the installers mean the RHDH Operator, can we use this stmt: configuring the PostgreSQL databases by using the {product} Operator is not intended for production use?

Reply:
Installers mean both Operator and Helm Chart I think.
To me "configuring PostgreSQL databases... " is confusing because there are some configuration piece in both cases. 
The difference is that in "local database" case our installer (Operator/Chart) installs/uninstalls/defines credentials for this database and for "external database" just configures given credentials.

Reply:
explicitly mentioned that it for configuring an external postgresql instance.